app/common/EnrolmentCommon.scala

```diff
@@ -19,9 +19,11 @@ package common
 object EnrolmentKeys {
   val Individual = "HMRC-MTD-IT"
   val Agent = "HMRC-AS-AGENT"
+  val nino = "HMRC-NI"
```

Comment:
Is this enrolment key correct?

Reply:
Yep, auth see enrolment keys differently to enrolment store, it should be the same key across all of these PRs.

Reply:
Ok sure, I've confirmed this is correct